x11-misc/grun/grun-0.9.3-r2.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A GTK based Run dialog resembling the Windows Run dialog, just like xexec"
+HOMEPAGE="https://github.com/lrgc/grun"
+SRC_URI="https://grun.googlecode.com/files/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="nls"
+
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+:2
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-no_nls.patch
+	"${FILESDIR}"/${P}-gcc14.patch
+)
+
+src_configure() {
+	[[ -z ${TERM} ]] && TERM=xterm
+
+	econf \
+		$(use_enable nls) \
+		--disable-gtktest \
+		--enable-associations \
+		--enable-testfile \
+		--with-default-xterm=${TERM}
+}
+
+src_install() {
+	default
+	dodoc AUTHORS BUGS ChangeLog NEWS README TODO
+}
```

Comment:
i think this could be removed entirely. portage should per default install all these doc files.